<!-- launch file to start sensoray capture device-->
<launch>
    <!-- <param name="use_sim_time"  value="true"/> -->
    <node name="sensoray_cam_stereo" pkg="sensoray_cam" type="sensoray_cam_node" output="screen">
		<!-- b_size=2 : 320*240 
			 b_size=4 : 640*480 -->		
		<param name="b_size"  value="2"/> 
        <param name="video_device1"   value="/dev/video0"/>
        <param name="video_device2"   value="/dev/video1"/>
        <param name="camera_info_url1" value="file://$(find sensoray_cam)/param/calib/wide_left.yaml"/>
        <param name="camera_info_url2" value="file://$(find sensoray_cam)/param/calib/wide_right.yaml"/>
    </node>
</launch>
